apps/builder/app/builder/features/style-panel/sections/backgrounds/backgrounds.tsx

```diff
@@ -117,6 +125,75 @@ const Layer = (props: {
   );
 };
 
+export const BackgroundsCollapsibleSection = (
+  props: CollapsibleSectionProps
+) => {
+  const { label, children } = props;
+  const [isOpen, setIsOpen] = useOpenState(props);
+
+  const layersStyleSource = getLayersStyleSource(
+    props.categoryProps.currentStyle
+  );
+  const dots: ("local" | "remote")[] = [];
+
+  if (layersStyleSource === "local" || layersStyleSource === "remote") {
+    dots.push(layersStyleSource);
+  }
+
+  return (
+    <CollapsibleSectionBase
+      label={label}
+      fullWidth
+      isOpen={isOpen}
+      onOpenChange={(nextIsOpen) => {
+        setIsOpen(nextIsOpen);
+      }}
+      trigger={
+        <SectionTitle
+          dots={dots}
+          suffix={
+            <SectionTitleButton
+              prefix={<PlusIcon />}
+              onClick={() => {
+                const { currentStyle, createBatchUpdate } = props.categoryProps;
+                addLayer(currentStyle, createBatchUpdate);
+
+                setIsOpen(true);
+              }}
+            />
+          }
+        >
+          <PropertyName
+            style={props.categoryProps.currentStyle}
+            property={layeredBackgroundProps}
+            label={
+              <SectionTitleLabel
+                color={layersStyleSource}
+                onClick={(event) => {
+                  // This code ensures that the onReset callback from PropertyName is triggered without closing the section.
+                  // To achieve this, stopPropagation is used instead of preventDefault.
+                  // Using preventDefault would prevent the Reset trigger from being triggered as well.
+                  event.stopPropagation();
```

Comment:
I thought about but stopPropagation is so so solution that I decided to have it explicitly in exact place its needed. As here is mostly issue because of PropertyName